src/nuklear_button.c

```diff
@@ -108,7 +110,7 @@ nk_draw_button(struct nk_command_buffer *out,
 }
 NK_LIB int
 nk_do_button(nk_flags *state, struct nk_command_buffer *out, struct nk_rect r,
-    const struct nk_style_button *style, const struct nk_input *in,
+    const struct nk_style_button *style, struct nk_input *in,
```

Comment:
Removing `const` here (an other places) is a breaking change in cases where people use `-Werror`, because users might have written code simular to this:
```c
const struct nk_input *in = ...
...
nk_do_button(state, out, r, style, in, behavior, content)
```

Which would now throw this error:
```
error: passing 'const struct nk_input *' to parameter of type 'struct nk_input *' discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
```

Should the major version be incremented because of this?

Reply:
I agree but it's the only straightforward way to fix it.

Reply:
Yeah, the missing `const`s are a pity, but I also don't see any more suitable way to solve it. Maybe we should increase the *minor* part of the version instead of the *bugfix* part to emphasize this breaking change (but in the context of the usage of Nuklear, I think this does not deserve a *major* version bump).

Reply:
Hmm. Yea bumping major really is overkill, even though it is consistent with the documentation for the changelog. I think minor bump + emphasis is good enough.